wiki/newcomers-FAQ.md

```diff
@@ -0,0 +1 @@
+# Newcomers FAQ
```

Comment:
Should it be empty?

Reply:
It's a proposition that such file should exist, but we yet have to discuss it. I'll delete it for now.